environment.yml

```diff
@@ -4,10 +4,17 @@ channels:
   - nodefaults
 dependencies:
   - cupy~=13.3.0
+  - dask-jobqueue~=0.9.0
+  - rapidsai-nightly::kvikio>=25.04.00a
```

Comment:
FWIW KvikIO 25.04 should be released later this week

ref: https://docs.rapids.ai/maintainers/

Reply:
Yes, I'm keeping a close eye on that release, and will update this once it's out!